Packages/Newtonsoft.Json-for-Unity.Converters/Configuration/KeyedConfig.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Newtonsoft.Json.UnityConverters.Configuration
+{
+#pragma warning disable CA2235 // Mark all non-serializable fields
+    [Serializable]
+    public struct KeyedConfig : IEquatable<KeyedConfig>
+    {
+        public string key;
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5865055)